graylog2-web-interface/src/views/stores/SearchStore.ts

```diff
@@ -55,6 +55,7 @@ export type SearchStoreState = {
   search: Search,
   result: SearchResult,
   widgetMapping: WidgetMapping,
+  filterSearchTypes: FilterSearchTypes | undefined | null;
```

Comment:
Maybe we do want to limit this to `FilterSearchTypes | null`? `null` would be used if it should not be filtered at all. This way we could ensure that `undefined` is not passed accidentally due to a programmatic error here (e.g. an optional chain that is short-circuiting).